Umbraco Cloud Release notes for February (#5891)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/2024-02-releasenotes.md

## Project History Page (Beta)
The "Project History Page," today released as a beta release, provides a detailed insight into the significant activities within your project. This feature meticulously catalogs events initiated by users or by Umbraco Cloud itself, including modifications to your project plan, portal deployments, as well as both automatic and manual upgrades.

![Project History Page](images/ProjectHistory.gif)

Each entry in the overview is interactive, allowing you to click on it for more detailed information. This feature ensures that you have easy access to in-depth details about each event, further enhancing your insight into your project's history and ongoing activities.

The overview today concentrates on activities spurred by users and Umbraco Cloud. However, our vision for the future is more expansive. As we diligently work on enhancing and refining this feature, the "Project History Page" is set to evolve into an all-encompassing overview. It will offer an unparalleled glimpse into the entirety of events that have transpired throughout your cloud project's existence.

## New Upgrade Details Page

> The new "Upgrade Details" page is your gateway to detailed insights for every upgrade. By clicking the 'info' link from the "Project History" page for a manual or automatic upgrade, you'll land on a page that not only highlights the essentials—product version, precise timestamps, current statuses, and error reports for troubleshooting—but also showcases the impact of these upgrades across all relevant environments for your project.

Reviewer: ```suggestion
The new "Upgrade Details" page is your gateway to detailed insights for every upgrade. By clicking the 'info' link from the "Project History" page for manual or automatic upgrades, you'll land on a page that highlights the essentials—product version, precise timestamps, current statuses, and error reports for troubleshooting. The page also showcases the impact of these upgrades across all relevant environments for your project.
```

![Upgrade Details Page](images/UpgradeDetailsPage.gif)

The "Upgrade Details" page offers a clear view of how updates are applied within each environment of your project. This feature provides a straightforward breakdown, enabling you to understand the specific effects and application of upgrades across different environments. Designed to enhance your workflow, this page equips you with essential knowledge for efficient upgrade management, supports precise decision-making, and contributes to the smooth operation and success of your project.

We're here to support your journey every step of the way. Should you have any questions or need further assistance, feel free to reach out. Happy upgrading!

## Improved Search on Projects Dashboard

> Finally, we have rolled out a significant update to the search functionality on the Projects Dashboard, making it easier and faster for you to navigate your projects. Now, you're not limited to searching by project names. This enhancement introduces the ability to search using a wide array of associated tags and even specific product versions within each project's environment. Examples of searchable tags include project type, project plan, whether the project is a baseline or a child, and payment type, providing a multifaceted approach to organizing and locating your projects.

Reviewer: ```suggestion
Finally, we have rolled out a significant update to the search functionality on the Projects Dashboard. The update makes it easier and faster for you to navigate your projects. Now, you're not limited to only searching by project names. This enhancement introduces the ability to search using a wide array of associated tags and even specific product versions within each project's environment. 

Examples of searchable tags include:

* Project type.
* Project plans.
* Payment type. 
* If the project is a baseline or a child.

Providing a multifaceted approach to organizing and locating your projects.
```
